Doc/library/typing.rst

```diff
@@ -1222,6 +1228,33 @@ These can be used as types in annotations. They all support subscription using
    is allowed as type argument to ``Literal[...]``, but type checkers may
    impose restrictions. See :pep:`586` for more details about literal types.
 
+   ``Literal`` is very similar to :class:`Union`, the main difference is that its
```

Comment:
This sentence is likely to cause confusion for many readers. Literal and Union are very different things in the type system.

Reply:
Would it be better as "The Literal annotation supports the same operations as Union"?

Reply:
I think it's better not to draw comparisons. Literal and Union have similar runtime behavior but they mean different things, and I don't think it will help most readers to compare them.

Reply:
Fair enough